lib/node_modules/@stdlib/stats/meanpw/lib/main.js

```diff
@@ -0,0 +1,101 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2025 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MODULES //
+
+var dtypes = require( '@stdlib/ndarray/dtypes' );
+var gmeanpw = require( '@stdlib/stats/base/ndarray/meanpw' );
+var dmeanpw = require( '@stdlib/stats/base/ndarray/dmeanpw' );
+var smeanpw = require( '@stdlib/stats/base/ndarray/smeanpw' );
+var factory = require( '@stdlib/ndarray/base/unary-reduce-strided1d-dispatch-factory' );
+
+
+// VARIABLES //
+
+var idtypes = dtypes( 'real_and_generic' );
+var odtypes = dtypes( 'real_floating_point_and_generic' );
+var policies = {
+	'output': 'real_floating_point_and_generic',
+	'casting': 'none'
+};
+var table = {
+	'types': [
+		'float64', // input
+		'float32' // input
+	],
+	'fcns': [
+		dmeanpw,
+		smeanpw
+	],
+	'default': gmeanpw
+};
+
+
+// MAIN //
+
+/**
+* Computes the arithmetic mean along one or more ndarray dimensions using pairwise summation.
+*
+* @name mean
```

Comment:
```suggestion
* @name meanpw
```

Remember to update this alias.